internal/xds/bootstrap/bootstrap.go

```diff
@@ -113,8 +126,12 @@ func GetRenderedBootstrapConfig(proxyMetrics *egcfgv1a1.ProxyMetrics) (string, e
 	var (
 		enablePrometheus bool
 		metricSinks      []metricSink
+		StatsMatcher     StatsMatcherParameters
 	)
 
+	// Set default envoy proxy Stats
```

Comment:
why are we adding all existing ones ?
shouldnt be leave this field unset if `proxyMetrics.Matches` is nil and set it to `matches` if not nil ?

im guessing the need for this feature as a user is - "hey EG, you've enabled all these stats by default, I'd like to limit it to specific stats to reduce storage $$$"

Reply:
You are right，when match is not empty, the default metric field should be added.
gateway enables all these stats by default.